rustotp/src/lib.rs

```diff
@@ -1,41 +1,13 @@
 extern crate rusthotp;
+use rusthotp::{HotpOutput, hotp};
 
-use rusthotp::{ HotpOutput, hotp };
+extern crate byteorder;
+use byteorder::{BigEndian, ByteOrder};
 
```

Comment:
I was sloppy with a rebase, which I'm pretty sure threw away a lot of stuff. I probably need to use the reflog to salvage and remerge.